docs/content/fantomas-setting-icon.js

```diff
@@ -0,0 +1,28 @@
+import { html } from 'https://cdn.skypack.dev/lit';
+import { component } from 'https://cdn.skypack.dev/haunted';
+
+
+function FantomasSettingIcon({ tooltip, type }) {
+    let settingType = 
+        type === 'green' ? {icon: "bi-check-circle-fill", color: "green-recommendation", tooltip: tooltip?tooltip:"This setting is good to use"}:  
+        type === 'orange' ? {icon:"bi-exclamation-circle-fill", color:"orange-recommendation", tooltip: tooltip?tooltip:"This setting is not recommended"}:
+        type === 'red' ? {icon: "bi-x-circle-fill", color:"red-recommendation", tooltip: tooltip?tooltip:"You shouldn't use this setting"}:
+        null
+
+    return  html`<i class="bi ${settingType.icon} ${settingType.color}" 
+                    data-bs-toggle="tooltip" data-bs-custom-class="${type}-tooltip" data-bs-title="${settingType.tooltip}"></i>`;
+}
+
+function FantomasSettingIconGResearch({ tooltip }) {
+    let root = window.fsdocs_search_baseurl
```

Comment:
Why don't you use the data-root attribute from html element?
That is slightly less worse than the using the global variable.